COPYING-LGPL

```diff
@@ -0,0 +1,175 @@
+Files in this code repository are to be licensed under the LGPLv3, which you'll
```

Comment:
In the files `COPYING` and `README.md` we're using the terms `LGPL-3.0-or-later` and `GPL-3.0-or-later` whereas here we're using `LGPLv3` and `GPLv3`. This usage is inconsistent IMHO. I don't know which is the better / more correct way to go, but let's just be consistent ... but if I had to pick I'd use the terms from the other files.